src/swarm/neo/request/RequestEventDispatcher.d

```diff
@@ -821,8 +826,17 @@ public struct RequestEventDispatcher
     {
         while ( this.queued_signals.length > 0 )
         {
-            auto code = this.queued_signals[0];
-            Array.removeShift(this.queued_signals, 0);
+            auto code = this.queued_signals.array()[0];
+
+            // Shift all the elements to the left by one
+            void* src = this.queued_signals.array().ptr + 1;
```

Comment:
`this.queued_signals.array().ptr + ubyte.sizeof`? (For consistency.)

Reply:
It's actually more consistent now. Imagine replacing `queued_signals` with `int` array - in that case suggested code would move the `src` pointer four elements after the first. The existing code, however, always move the pointer to one past the first element (or one past the array, if it has only one element - which is guaranteed to have by the bound checked array access in the previous line).

Reply:
Ahh yes, I was thinking of `this.queued_signals.array().ptr + 1` as already operating on ` void*`.